atomic_reactor/plugins/build_orchestrate_build.py

```diff
@@ -229,42 +282,35 @@ def get_cluster_info(self, cluster, platform):
 
         conf = Configuration(**kwargs)
         osbs = OSBS(conf, conf)
-        try:
-            current_builds = self.get_current_builds(osbs)
-        except OsbsException as e:
-            # If the build is canceled reraise the error
-            if isinstance(e.cause, BuildCanceledException):
-                raise e
 
-            self.log.exception("Error occurred while listing builds on %s",
-                               cluster.name)
-            return ClusterInfo(cluster, platform, osbs, self.UNREACHABLE_CLUSTER_LOAD)
+        current_builds = self.get_current_builds(osbs)
```

Comment:
Does this want to be `with osbs.retries_disabled()`?

Reply:
self.get_current_builds has retries disabled L273